README.rst

```diff
@@ -4,7 +4,7 @@
 
 ----
 
-.. image:: https://github.com/crate/crate/workflows/CrateDB%20SQL/badge.svg?branch=master
+.. image:: https://github.com/crate/crate/actions/workflows/main.yml/badge.svg
```

Comment:
Maybe also use the `branch` parameter?
```suggestion
.. image:: https://github.com/crate/crate/actions/workflows/main.yml/badge.svg?branch=master
```

Reply:
I think it uses the default branch if not specified, which is master anyway